lib/rules/no-get.js

```diff
@@ -2,11 +2,19 @@
 
 const types = require('../utils/types');
 const emberUtils = require('../utils/ember');
+const assert = require('assert');
 
-function makeErrorMessageForGet(property, isImportedGet) {
-  return isImportedGet
-    ? `Use \`this.${property}\` instead of \`get(this, '${property}')\``
-    : `Use \`this.${property}\` instead of \`this.get('${property}')\``;
+function makeErrorMessageForGet(property, isImportedGet, useOptionalChaining) {
```

Comment:
Could you change this to `makeErrorMessageForGet(property, { isImportedGet, useOptionalChaining })`? Passing in multiple unnamed boolean parameters can become tricky.

Reply:
Good idea, fixed.